scene/resources/packed_scene.cpp

```diff
@@ -238,16 +238,37 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 
 					if (nprops[j].name & FLAG_PATH_PROPERTY_IS_NODE) {
 						uint32_t name_idx = nprops[j].name & (FLAG_PATH_PROPERTY_IS_NODE - 1);
+						NodeData::Property nprop = nprops[j];
+						Variant prop_variant = props[nprop.value];
+						StringName prop_name = snames[name_idx];
```

Comment:
```suggestion
						const StringName &prop_name = snames[name_idx];
```
(technically applies to other variables too)